COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Public Slack](#public-slack)
```

Comment:
We're defaulting to Public, yes? So shouldn't this just be [Slack](#slack)

Reply:
:+1: